web/public/mgmt/config.yml

```diff
@@ -3064,24 +3065,21 @@ collections:
       preview: true
     fields:
       - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false }
-      - {
-          label: "Industry Id",
-          name: "urlSlug",
-          widget: "relation",
-          collection: "roadmaps",
-          search_fields: ["{{name}}"],
-          options_length: 500,
-          required: true,
-          value_field: "{{id}}",
-          display_fields: ["{{name}}"],
-        }
```

Comment:
[boulder] We still need a slug. We just need to de-couple this from the industry id